test/serializers/attributes_test.rb

```diff
@@ -23,12 +23,6 @@ def test_attributes_with_fields_option
                      @profile_serializer.attributes(fields: [:name]))
       end
 
-      def test_required_fields
-        assert_equal({name: 'Name 1', description: 'Description 1'},
-                     @profile_serializer.attributes(fields: [:name, :description], required_fields: [:name]))
-
-      end
-
```

Comment:
What you imply is true, and the answer is no, it is not possible anymore. However, this option is not used anywhere inside of AMS, and I am quite unsure as to why one would use it outside considering its only use was to add attributes without declaring them as attributes. I'm convinced it was created as a hack for the sole purpose of handling the `json_api_type` and `id`.